src/Html.php

```diff
@@ -1426,7 +1426,7 @@ public static function getMenuInfos()
                 'title' => __('Administration'),
                 'types' => [
                     'User', 'Group', 'Entity', 'Rule',
-                    'Profile', 'QueuedNotification', 'Glpi\\Event', 'Glpi\Inventory\Inventory'
+                    'Profile', 'QueuedNotification', 'PrinterCartridgeLevelAlert', 'Glpi\\Event', 'Glpi\Inventory\Inventory'
```

Comment:
I'm not sure having this directly in the menu is a good idea.

Reply:
This is the behavior introduced by the AdditionalAlerts plugin. Having a summary page that allows to have a list of installed cartridges in a low level state is very usefull IMHO.

Reply:
Hi, I'm back to this PR... I'm really not sure about the place Alerts should be put into... Printers cartridges alerts are not related to GLPI administration, they are representing much more a supervision tool like Tools/Reports... Do I miss something?